scripts/typescript_checks.py

```diff
@@ -178,6 +177,9 @@
     'core/templates/filters/format-timer.filter.ts',
     'core/templates/filters/parameterize-rule-description.filter.spec.ts',
     'core/templates/filters/parameterize-rule-description.filter.ts',
+    'core/templates/filters/parameterize-rule-description.pipe.spec.ts',
+    'core/templates/filters/parameterize-rule-description.pipe.ts',
+    'core/templates/filters/remove-duplicates-in-array.pipe.spec.ts',
```

Comment:
New files should not be added to this exclusion list without approval from @vojtechjelinek (see the comment at the top of the list)

Reply:
@vojtechjelinek, @JeeveshGarg is going to type `parameterize-rule-description.filter.ts` same  type errors are in `parameterize-rule-description.pipe.ts` file solving one will make automatically solve another + interaction type needs to be corrected so I am adding these two files in the list,

Reply:
I will approve once @JeeveshGarg confirms these changes are fine with him.

Reply:
@vojtechjelinek while reviewing typescript_checks.py things we need to take care are:

- @heyimShivam is allowed to add new Angular files, as we have lack of files for improve frontend types. (already discuss this with green light reviewer).
- Files removed by him must be AngularJs files (rename in most cases) OR dead code.
- Newly added files must be edited by Shivam, they may be dublicate files, as we need AngularJs & Angular both files somewhere.

Hence LGTM.

Thanks.

**cc:** @U8NWXD .

Reply:
Okay, I'm fine with that.